core/src/main/java/com/microsoft/applicationinsights/internal/schemav2/AvailabilityData.java

```diff
@@ -233,16 +233,16 @@ protected void serializeContent(JsonTelemetryDataSerializer writer) throws IOExc
         super.serializeContent(writer);
         writer.write("ver", ver);
         
-        writer.write("id", id);
+        writer.write("id", id, 64, true);
```

Comment:
Alternatively to `true` or `false` you can have two methods - one `write` and another `writeRequired`. This way you save one runtime check and code a bit more readable

Reply:
@SergeyKanzhelev done. Thanks for the input!